app/helpers/repositories_helper.rb

```diff
@@ -44,6 +44,17 @@ def access_options
   end
 
   def repository_modal_body
-    modal_body(t("delete_repository"), t("delete_repository_desc"), (controller_name == "repositories" ? resource : parent), t("repository.delete"))
+    modal_body(t('delete_repository'), t('delete_repository_desc'),
+      reource_repository, t('repository.delete'))
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.